web/source/osk/embedded/subkeyDelegator.ts

```diff
@@ -32,31 +53,29 @@ namespace com.keyman.osk.embedded {
           // Handle selection of base key underneath the subkey array.
           keyEvent = this.vkbd.keyEventFromSpec(this.baseKey.key.spec as keyboards.ActiveKey, null);
           this.baseKey.key.highlight(false);
-        } else {
+        } else if(keyCoreID != null) {
```

Comment:
if `keyCoreID == null` then we just default to `this.resolver(keyEvent)`, i.e. `this.resolver(null)`? I wonder if we should comment that; I know I can figure it out but it's a little tricky to follow.